examples/plot_newfitting_Tgas.py

```diff
@@ -61,8 +59,8 @@
     "wmax": 1050,  # Ending wavelength/wavenumber for the cropping range.
     "wunit": "cm-1",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
     "mole_fraction": 0.01,  # Species mole fraction, from 0 to 1.
-    "pressure": 10,  # Partial pressure of gas, in "bar" unit.
-    "path_length": 1,  # Experimental path length, in "cm" unit.
+    "pressure": 10,  # Partial pressure of gas, in "bar" unit by default, but you can also use Astropy units.
```

Comment:
Warning. This the total pressure not the partial pressure.

Correct this everywhere